packages/amplify-e2e-core/src/init/amplifyPush.ts

```diff
@@ -147,6 +147,31 @@ export function amplifyPushWithoutCodegen(
   });
 }
 
+export function amplifyPushSecretsWithoutCodegen(
+  cwd: string,
+  testingWithLatestCodebase: boolean = false,
+  allowDestructiveUpdates: boolean = false,
```

Comment:
I don't think this is needed here

Reply:
removed `allowDestructiveUpdates` as this is not used with GraphQL api.